src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -1,11 +1,224 @@
 package frc.team2412.robot.subsystem;
 
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import static frc.team2412.robot.subsystem.IndexSubsystem.IndexConstants.*;
+import static frc.team2412.robot.subsystem.IndexSubsystem.IndexConstants.IndexMotorState.*;
+import static frc.team2412.robot.subsystem.IndexSubsystem.IndexConstants.IndexPositionState.*;
 
+import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.util.MultiplexedColorSensor;
+import io.github.oblarg.oblog.annotations.Log;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
 
 public class IndexSubsystem extends SubsystemBase {
-    public IndexSubsystem(WPI_TalonFX indexMotor) {
 
+    // Constants
+
+    public static class IndexConstants {
+
+        public static Alliance teamColor = DriverStation.getAlliance();
+
+        // Color Sensor values
+
+        public static Color BLUE_CARGO_COLOR = new Color(0, 0, 1);
+        public static Color RED_CARGO_COLOR = new Color(1, 0, 0);
```

Comment:
These aren't used either - can they be deleted, or is code missing that should use them?